specification/metrics/listcost.md

```diff
@@ -0,0 +1,34 @@
+# List Cost
+
+The List Cost represents a charge inclusive of the impacts of free-tier, volume/tier-based, BYOL-based and dynamically priced SKU rates, while excluding any other discounts or the amortization of upfront charges (one-time or recurring). This cost is denominated in the [Billing Currency](#billingcurrency). It is used for calculating savings based on rate optimization activities, by comparing it with [Effective Cost](#effectivecost). In this context, activities such as increased consumption leading to lower volume/tier-based rates, the use of pre-owned software licenses (BYOL - Bring Your Own License), leveraging interruptible resources and/or services, or optimizing usage to take advantage of dynamic pricing models are not considered rate optimization activities.
```

Comment:
Trying to avoid generic references like "this" and "it". Also replacing the statement about what's in or not in rate optimization. May want to tweak this based on Udam's ListUnitPrice comments.
```suggestion
The List Cost represents a charge inclusive of the impacts of free-tier, volume/tier-based, BYOL-based and dynamically priced SKU rates, while excluding any other discounts or the amortization of upfront charges (one-time or recurring). List Cost is denominated in the [Billing Currency](#billingcurrency) and is used for calculating savings based on rate optimization activities, by comparing it with [Effective Cost](#effectivecost). List Unit Price cannot be used to calculate savings based on volume/tier-based pricing, the use of pre-owned software licenses (BYOL - Bring Your Own License), leveraging interruptible resources and/or services, or optimizing usage to take advantage of dynamic pricing models.
```

Reply:
I didn't propose a change for this, but just to add a note, I do think the first sentence feels overly complicated. It feels like we can simply say it's the "public, retail price for the SkuPriceId". I'm assuming that's the basis for Udam's simplification.

Reply:
I agree with Michael

Reply:
- New version of the definition is based on suggested changes.
- The sentence, "_List Unit Price cannot be used to calculate savings based on..._", was relocated to the supporting content and the reference to  'List Unit Price' was replaced with the reference to the 'List Cost'.